TrainingExtensions/onnx/src/python/aimet_onnx/meta/connectedgraph.py

```diff
@@ -519,9 +519,10 @@ def _link_branch_op_to_multiple_ops(self, branch_op: Op, product_list: list,
                 del self._products[product.name]
             else:
                 for output in self.model.graph.output:
-                    if output.name in product.name:
+                    if product.name.startswith(output.name + '_to_') or \
```

Comment:
Is this a robust change - assuming the names to follow these '_to_' substrings? @quic-mtuttle - please advise.

Reply:
Yes, the connected graph product names should always follow this naming convention based on how we define them:

https://github.com/quic/aimet/blob/37f2cee1ae362eac1c814d5ff8281c5532a7d6ed/TrainingExtensions/onnx/src/python/aimet_onnx/meta/connectedgraph.py#L285-L287